haystack/document_stores/weaviate.py

```diff
@@ -571,7 +571,8 @@ def get_document_count(
 
```

Comment:
on line 567 and 670 you could use `.with_meta_count()` instead of `.with_fields("meta { count }")`.